packages/ui-react/src/containers/AdyenInitialPayment/AdyenInitialPayment.tsx

```diff
@@ -73,6 +77,8 @@ export default function AdyenInitialPayment({ setUpdatingOrder, type, setPayment
 
         await accountController.reloadActiveSubscription({ delay: 2000 });
 
+        // Todo: double check if this is the correct placement.
```

Comment:
Let's double check and remove the todo? :)

Reply:
Thanks for the reminder!